env.js

```diff
@@ -169,14 +169,6 @@ function config ( ) {
   env.pushover_api_token = readENV('PUSHOVER_API_TOKEN');
   env.pushover_user_key = readENV('PUSHOVER_USER_KEY') || readENV('PUSHOVER_GROUP_KEY');
 
-  // TODO: clean up a bit
```

Comment:
We'd like to keep everything that reads from env vars in `env.js`, then for testing it's easy to create a mocked env